CIP-0187? | Utilization-Scaled Pledge Bonus (#1193)
cardano-foundation/CIPs: CIP-0187/README.md

### Optional governance-controlled blend (phased adoption)

To enable gradual phase-in, governance MAY introduce a blending coefficient `b ∈ [0, 1]` such that:

```
A_blend = A_current · (1 − b + b · u)
```

`b = 0` reproduces today's behavior exactly. `b = 1` produces the full proposal as specified above. Intermediate values are valid. The blend coefficient, if adopted, is a new governance-settable protocol parameter; if `b` is not adopted, the change is a one-step replacement of `A_current` with `A_new` at a hard-fork boundary.

The body of this CIP analyzes the limit case `b = 1`. Implementors MAY choose to introduce the parameter or to switch directly; this CIP does not mandate either path.

### On-chain data structures

This proposal introduces **no new on-chain data, no new transaction types, no new certificates, and no new ledger state**. `S`, `P`, and `S_sat` are already snapshotted per epoch as part of the existing reward calculation pipeline. No CDDL schema changes are required. The Specification change is local to the formula used during reward calculation at the epoch boundary.

### Versioning

> This proposal is a single, atomic change to the reward calculation, taking effect at the hard-fork combinator event that activates it. The protocol parameter `a₀` continues to be versioned by the existing governance mechanism. If the optional blending coefficient `b` is adopted, it is versioned identically to `a₀`. No additional versioning scheme is introduced by this CIP.

Reviewer: This looks fine to me: in fact it's a generous response to the request for _Versioning_ since some readers might not know the protocol parameters are implicitly versioned — or think they are only changeable by CIP — and this addresses that question.

- **Small honest SPOs:** competitive viability via delegation rather than personal capital. A small-pledge pool that fills to saturation earns the same per-ADA pledge bonus as a whale-pledge pool at saturation.
- **Every delegator:** the central win. Adding delegation to any pool monotonically increases per-delegator ROA. The dilution regime is structurally removed. Pool selection simplifies to "find pools with high `P · u / S` and low `F`."
- **Cooperatives (Pool Ranger and others):** structural bargaining power as a side effect of being any large delegator. No protocol-level recognition, no registry, no stake-credential consolidation required. Withdrawal becomes a publicly observable mathematical event, giving SPOs continuous incentive to retain delegators, not just attract them.
- **Decentralization:** MPO behavior is taxed at the formula level proportional to `N`. Combined with the unchanged `a₀`, Cardano's intended `k`-pool diversity becomes self-enforcing rather than aspirational.
- **Treasury:** governance-tunable. Either reserve depletes slightly slower, or a one-time `a₀` recalibration preserves today's emission exactly. Single governance vote either way.

> ### Comparison to other proposals

Reviewer: For the following methods (especially those defined in CIPs) it will be a courtesy — making review & consideration of your proposal more effective — if you _hyperlink_ to any of these methods that are defined elsewhere.  (GitHub links are preferred for CIPs.)

(also suggested for _References_ section)

Reviewer: Resolved in https://github.com/cardano-foundation/CIPs/pull/1193/changes/b7580aa889bc00e2cdeb35159d5e00320b2435b8#diff-4ffe445f8e72a6cee701a5a989aef62750d96c6296058bb612cad8ff305b49a5.

2. **Test vectors.** Publish a set of input/output vectors covering the boundary cases listed in Acceptance Criteria, computable by any independent implementation.
3. **Governance proposal.** Submit a CIP-1694 governance action to activate the formula change at the next hard-fork combinator event. The action SHOULD specify whether the optional blending coefficient `b` is adopted; if so, its initial value.
4. **Wallet and tooling notice.** Coordinate with major wallet teams (Eternl, Lace, Nami, Yoroi, Flint) and explorers (Cexplorer, pool.pm, ADAPools, PoolTool) to update displayed ROA projections at the hard-fork epoch.
5. **Optional treasury recalibration.** If governance wishes to preserve average-utilization emission at today's level, a follow-up `a₀` recalibration vote MAY be submitted, with the target `a₀` computed from chain data immediately prior to the hard fork.
6. **Activation.** Hard-fork combinator activates the new formula at the chosen epoch.

Implementors have not yet committed; the `Implementors:` field is empty pending review by the ledger team.

> ## References

Reviewer: (would benefit from CIP & external links as also suggested for _Comparison to other proposals_ subsection)

Reviewer: Resolved in https://github.com/cardano-foundation/CIPs/pull/1193/changes/b7580aa889bc00e2cdeb35159d5e00320b2435b8#diff-4ffe445f8e72a6cee701a5a989aef62750d96c6296058bb612cad8ff305b49a5.

- [CIP-0050](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0050) — Pool leverage cap proposal (related; addresses a different symptom).
- Brünjes, Kiayias, Koutsoupias, Stouka — [*Reward Sharing Schemes for Stake Pools*](https://eprint.iacr.org/2018/1145) (the original RSS paper formalizing the formula this CIP modifies).
- [Interactive simulator](https://johnshearing.github.io/pool_ranger/CIP_UTILIZATION_SCALED_PLEDGE_BONUS.html) — adjustable parameters with four guided tutorials: (1) the math explainer, (2) who benefits and by how much, (3) the multi-pool operator problem, (4) free-form exploration. Source repository: [johnshearing/pool_ranger](https://github.com/johnshearing/pool_ranger).

## Acknowledgements

This proposal was drafted with assistance from Claude (Anthropic). The mathematical derivation, the constraint that `a₀` must be preserved, the proposed `u` scaling, and the analysis of cooperative-delegation effects emerged from extended dialog between the author and the model. All claims have been independently re-derived by the author. Any errors are the author's.

## Appendix A: Detailed Derivation and Worked Examples

> 

Reviewer: ```suggestion
## Appendices

### Appendix A: Detailed Derivation and Worked Examples

```
Our validator — not being also AI-based 😅 — relies on text match to know what each major section is & to confirm the basic document order.  I think this change will work to pass validation (so would the main section title `Appendix`) but also keep in mind:
* it's considered a breach of ordered lists & outline structure to have an `A` without a `B`; a `1` without a `2`, etc... and therefore according to general document style the `A` should be dropped;
* the numbers in general are bad because they break incoming links to Markdown sections whenever the number gets changed (e.g. when an AI ghostwriter might reorder, add, or drop sections);
* in any case you'll also have to change the incoming link to this section a couple times in the document above.

Reviewer: @johnshearing the validation is still failing after https://github.com/cardano-foundation/CIPs/pull/1193/commits/c32d3c6e5f61215aa7bb6d4f13d980fcb5583f8a because the top-level (H2, or `##`) heading is still your arbitrary long title instead the standard section titles `Appendix` or `Appendices`... please check again the suggestion above & try to do something similar.

Reviewer: Looks good to the validator now ✅

---
CIP: 187
Title: Utilization-Scaled Pledge Bonus
Category: Ledger
Status: Proposed
Authors:
    - John Shearing <johnshearing@gmail.com>
Implementors: []
Discussions:
    - Original PR: https://github.com/cardano-foundation/CIPs/pull/1193
    - Forum: https://forum.cardano.org/t/cip-utilization-scaled-pledge-bonus-the-multi-pool-buster/154614
Created: 2026-05-15
License: CC-BY-4.0
---

## Abstract

> Cardano's reward formula contains a structural defect in the pledge bonus term `A`: under the current specification, `A = r · a₀ · P / (1 + a₀)` depends on the operator's pledge `P` but not on the pool's total stake `S`. As delegation arrives at a high-pledge pool, the per-delegator share `(A − F)/S` strictly falls, so adding stake dilutes everyone already there. The very behavior the protocol's Nash equilibrium assumes — rational delegators concentrating stake into high-pledge, well-operated pools — is punished by the formula itself.

Reviewer: ```suggestion
Cardano's reward formula contains a potential weakness in the pledge bonus term `A`: under the current specification, `A = r · a₀ · P / (1 + a₀)` depends on the operator's pledge `P` but not on the pool's total stake `S`. As delegation arrives at a high-pledge pool, the per-delegator share `(A − F)/S` strictly falls, so adding stake dilutes everyone already there. The very behavior the protocol's Nash equilibrium assumes — rational delegators concentrating stake into high-pledge, well-operated pools — is punished by the formula itself.
```
Leading in with the term `structural defect` might produce more controversy than consideration of your proposal itself: by shifting emphasis toward defence of the original RSS conception.

By leading in with a less emphatic proposition, the writing would immediately afterward provide some justification for the reader to make such a judgement and perhaps to conclude on their own that there may be a "defect".

This CIP proposes a parameter-free reshape of the existing pledge bonus by multiplying it by pool utilization `u = min(S, S_sat) / S_sat`. The new bonus is `A = r · a₀ · P · min(S, S_sat) / (S_sat · (1 + a₀))`. A saturated pool earns exactly today's pledge bonus; a half-utilized pool earns half; an empty pledged pool earns nothing from pledge until delegators arrive. No new parameter is introduced — `a₀` retains its name, governance dial, and Sybil-resistance role. The change removes the dilution regime entirely, strengthens Sybil resistance at the formula level, makes "fill pools to saturation" self-enforcing at every utilization level, and gives small honest operators a viable on-ramp through delegation rather than personal capital.

## Motivation: Why is this CIP necessary?

> The standard Cardano delegator return formula (Reward Sharing Scheme, CIP-0084) reduces to:

Reviewer: ```suggestion
The standard Cardano delegator return formula (Reward Sharing Scheme) reduces to:
```
CIP-0084 defines the process of _updating_ the Ledger specification: but it _is not_ the Ledger specification: which in turn contains the reward sharing scheme.

It may also help the reader to link to the RSS document already included in your _References_ and/or include the [current Ledger `spec`](https://github.com/johnshearing/CIPs/tree/utilization-scaled-pledge-bonus/CIP-0084#ledger-eras) incidentally linked from CIP-0084 as one of those references.

I'll try to flag these inappropriate references to CIP-0084 as I find them & please also double check for these when updating.

4. **The Sybil deterrent leans entirely on the delegator-preference channel.** Splitting `X` ADA into `N` pools leaves the total pledge bonus unchanged under the current formula; only delegator avoidance makes the split unprofitable. The formula itself does no Sybil work beyond setting `A`'s magnitude.
5. **Empty pledged pools earn the same per-pledge bonus as filled pools.** The protocol pays for an entitlement that has not yet produced the equilibrium it is meant to incentivize.

The obvious direct fix — lowering `a₀` — is unacceptable because `a₀` carries the Sybil-resistance load. Removing or weakening it reopens the stake-splitting channel. Any acceptable fix must preserve `a₀`'s deterrent role while removing the dilution.

A fuller treatment — the derivation of the proposed formula, numerical worked examples, and game-theoretic analysis of `a₀` under both regimes — is in [Appendix A](#appendix-a-detailed-derivation-and-worked-examples) below. An [interactive simulator](https://johnshearing.github.io/pool_ranger/CIP_UTILIZATION_SCALED_PLEDGE_BONUS.html) with adjustable parameters and four guided tutorials accompanies the proposal.

## Specification

### Notation

> All symbols carry their standard CIP-0084 / Reward Sharing Scheme meaning:

Reviewer: ```suggestion
All symbols carry their standard Ledger Specification / Reward Sharing Scheme meaning:
```
Another example of where CIP-0084 should be removed.

2. **Test vectors.** Publish a set of input/output vectors covering the boundary cases listed in Acceptance Criteria, computable by any independent implementation.
3. **Governance proposal.** Submit a CIP-1694 governance action to activate the formula change at the next hard-fork combinator event. The action SHOULD specify whether the optional blending coefficient `b` is adopted; if so, its initial value.
4. **Wallet and tooling notice.** Coordinate with major wallet teams (Eternl, Lace, Nami, Yoroi, Flint) and explorers (Cexplorer, pool.pm, ADAPools, PoolTool) to update displayed ROA projections at the hard-fork epoch.
5. **Optional treasury recalibration.** If governance wishes to preserve average-utilization emission at today's level, a follow-up `a₀` recalibration vote MAY be submitted, with the target `a₀` computed from chain data immediately prior to the hard fork.
6. **Activation.** Hard-fork combinator activates the new formula at the chosen epoch.

Implementors have not yet committed; the `Implementors:` field is empty pending review by the ledger team.

## References

> - [CIP-0001](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001) — CIP process and document structure.

Reviewer: ```suggestion
```
This is implicit in the process followed by all CIPs & CPSs and so should be removed as redundant.

> - [CIP-0084](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0084) — Reward Sharing Scheme (the formula this CIP modifies).

Reviewer: ```suggestion
```
As above: CIP-0084 doesn't contain these things... also, the subject of how a Ledger CIP should be written (the actual purpose of CIP-0084) is out of the scope of your proposal.

> - [CIP-1694](https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694) — On-chain governance actions (the mechanism this CIP would be activated through).

Reviewer: The reference to `CIP-1694` could likewise be considered redundant but may also help academic audiences considering the mathematical proposition & who mightn't have heard of CIP-1694 before.  But even if so, it would be easier for them to see that term linked upon first use up in your _Path to Active_ instead.
